lib/nth_prime_cases.rb

```diff
@@ -0,0 +1,19 @@
+class NthPrimeCase < OpenStruct
+  def name
+    'test_%s' % description.downcase.gsub(/[ -]/, '_')
+  end
+
+  def do
```

Comment:
There's no documentation on `do`. I had seen it in a few other files when I was working on this.

I'll change it to `actual`.